node-require/calculate.js

```diff
@@ -0,0 +1,14 @@
+const add = require('./add');
+const sub = require('./subtract');
+const multiply = require('./multiply');
+const divide = require('./divide');
+
+if (process.argv[3] === 'add') {
+  console.log('result: ', add(Number(process.argv[2]), Number(process.argv[4])));
```

Comment:
It would make your code more consistent and easier to maintain if you converted the arguments to number up above, so all functions were being passed a number.